pom.xml

```diff
@@ -190,7 +189,7 @@
         <jaxb2basics.version>0.6.5.1</jaxb2basics.version>
         <jaxb2basics.plugin.version>0.6.4</jaxb2basics.plugin.version>
         <jdom.version>1.0</jdom.version>
-        <jgroups.version>3.6.13.Final</jgroups.version>
+        <jgroups.version>3.2.16.Final</jgroups.version>
```

Comment:
Good catch, I picked the wrong version when resolving a merge conflict, this has been updated.